clients/python/Makefile

```diff
@@ -24,8 +24,9 @@ deploy-test-minio:
 	kubectl port-forward -n minio svc/minio 9000:9000 &
 	
 .PHONY: test-e2e
-test-e2e: deploy-latest-mr deploy-test-minio
-	bash -c 'set -a && source ../../scripts/manifests/minio/.env && set +a && poetry run pytest --e2e -s -rA'
+test-e2e: deploy-latest-mr deploy-local-registry deploy-test-minio
+	@set -a; . ../../scripts/manifests/minio/.env; set +a; \
```

Comment:
@tarilabs can you verify whether this will work in non linux envs too?

Reply:
it does 
![image](https://github.com/user-attachments/assets/b172736d-359d-41b9-ba52-784a54bbcad9)